src/go/k8s/Makefile

```diff
@@ -73,11 +75,19 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build -f Dockerfile -t ${IMG} ../
+	docker build -f Dockerfile -t ${OPERATOR_IMG_LATEST} ../
 
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+docker-tag-release:
+	docker tag ${OPERATOR_IMG_LATEST} ${OPERATOR_IMG_RELEASE}
+
+# Push to dockerhub
+docker-push-release:
+	if curl --silent -f -lSL https://index.docker.io/v1/repositories/${OPERATOR_IMG}/tags | grep '"name": "$(shell cat VERSION)"' ; then \
+	  echo "Image ${OPERATOR_IMG_RELEASE} already exists, skipping push"; \
+	else \
```

Comment:
Should we also push the configurator image with the same tags? Cluster configuration relies on it.

Reply:
oh yeah, I forgot about that! Thank you!